src/test/java/teammates/test/cases/datatransfer/FeedbackSessionAttributesTest.java

```diff
@@ -27,10 +30,95 @@ public void classSetup() {
         startTime = TimeHelper.combineDateTime("09/05/2016", "1000");
         endTime = TimeHelper.combineDateTime("09/05/2017", "1000");
 
-        fsa = new FeedbackSessionAttributes(null,
-                null, null, null, null, startTime, endTime,
-                null, null, 8, 15, FeedbackSessionType.STANDARD,
-                false, false, false, false, false, false, false);
+        fsa = FeedbackSessionAttributes
+                .builder(null, null, null)
```

Comment:
Even though this is a test, required fields should not be null. If this does not throw errors, it may well be a sign that the "required" fields are not properly enforced.